apps/server/src/modules/board/controller/mapper/elements-response.mapper.ts

```diff
@@ -0,0 +1,16 @@
+import { AnyBoardDo, FileElement, TextElement } from '@shared/domain';
+import { AnyContentElementResponse } from '../dto';
+import { FileElementResponseMapper } from './file-element-response.mapper';
+import { TextElementResponseMapper } from './text-element-response.mapper';
+
+export class ElementsResponseMapper {
+	static mapToResponse(element: AnyBoardDo): AnyContentElementResponse {
```

Comment:
For know it is ok. If we get a lot more different elements we should switch it to a strategy.